src/app/app.css

```diff
@@ -153,6 +173,12 @@ html, body, #root {
   text-overflow: ellipsis;
 }
 
+.pf-c-label.clickable-label-hovered {
+  cursor: pointer;
+  --pf-c-label__content--before--BorderWidth: 2.5px;
+  --pf-c-label__content--before--BorderColor: #8a8d90;
```

Comment:
Wondering which visual effect for this css rule?

Reply:
On this note, maybe we can just remove it as their natural border color with with 2.5px looks better than the grey border :D